src/category_theory/adjunction/basic.lean

```diff
@@ -28,16 +28,20 @@ structure adjunction (F : C ⥤ D) (G : D ⥤ C) :=
 
 infix ` ⊣ `:15 := adjunction
 
+/-- A class giving a chosen right adjoint to the functor `left`. -/
 class is_left_adjoint (left : C ⥤ D) :=
```

Comment:
I don't know the category theory library, and not so relevant to this PR, but `is_left_adjoint` sounds like a prop. This makes the doc string read wrong to me even though it seems right.

Reply:
It's not a Prop, it does actually contain the data of the right adjoint - this is a debatable design choice but I think the docstring does describe what is going on here.

Reply:
Yup, the names in `category_theory/` sound `Prop`pier than they typically are...